tests/ordering/tests.py

```diff
@@ -480,6 +480,17 @@ def test_order_by_f_expression_duplicates(self):
             attrgetter("headline"),
         )
 
+    def test_order_by_f_expression_alias(self):
+        qs = (
+            Article.objects.filter(id__lt=0)
+            .annotate(test=Value(42))
+            .order_by(F("test").asc())
```

Comment:
`asc()` is unnecessary:
```suggestion
            .order_by(F("test"))
```